src/openrct2/game.c

```diff
@@ -1121,6 +1121,9 @@ void game_fix_save_vars()
 
     // Fix invalid vehicle sprite sizes, thus preventing visual corruption of sprites
     fix_invalid_vehicle_sprite_sizes();
+
+    // Fix gParkEntrance locations for which the map_element no longer exists
```

Comment:
Could you copy this comment to the function definition as well?

Reply:
Yes, of course. I thought I had.  Trying to get to bed, so was clearly editing too fast for my own good.